client/my-sites/plans-v2/products-grid-alt-2/index.tsx

```diff
@@ -156,7 +156,7 @@ const ProductsGridAlt2: React.FC< ProductsGridProps > = ( {
 					/>
 				) ) }
 
-				{ showJetpackFreeCard && siteId && (
```

Comment:
This is to fix a bug in `v2 - slide outs` which removes the Jetpack Free from the Pricing page.